pkg/distributor/distributor.go

```diff
@@ -26,7 +25,7 @@ import (
 	"github.com/cortexproject/cortex/pkg/util/flagext"
 	"github.com/cortexproject/cortex/pkg/util/limiter"
 	"github.com/cortexproject/cortex/pkg/util/validation"
-	billing "github.com/weaveworks/billing-client"
```

Comment:
Why did this change? I prefer the package being explicit.

Reply:
My IDE flagged it as "redundant alias" and suggested to remove it and I prefer to get rid of warnings when I can. So I did and nothing broke. I can put it back, if you want (and adjust my IDE warnings setting :))

Reply:
Please put it back :)

Reply:
It's back.